docs/blob-nfs.rst

```diff
@@ -1,11 +1,10 @@
-===========================================
```

Comment:
I think the adornments were already correct.

Reply:
Done.